personal_assistant/contacts/templates/contacts/detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends "pa_core/home.html" %} {% block title%}
+    {{contact.name}}
+{% endblock %}
+
+{% block content %}
+<p>Phone: <b>{{contact.phone}}</b></p>
+{% if contact.address %}
+<p>Address: <b>{{contact.address}}</b></p>
+{% endif %}
+{% if contact.email %}
+<p>E-mail: <b>{{contact.email}}</b></p>
+{% endif %}
+{% if contact.birthday %}
+<p>Birthday: <kbd>{{contact.birthday}}</kbd></p>
+{% endif %}
+<nav class="nav">
+    <ul>
+        <li><a href="/contacts" class="nav-link"> Go back to the contacts page </a></li>
```

Comment:
Краще не використовувати захардкоджені шляхи:
```suggestion
        <li><a href="{% url 'pa_contact:main' %}" class="nav-link">Go back to the contacts page</a></li>
```